src/components/schemas/file.yaml

```diff
@@ -56,10 +57,14 @@ file_out_embed:
       type: string
       enum:
         - INTEGRATION
+        - RECEIPT
```

Comment:
don't we need to add thumbnail and other types we support as well?

Reply:
other types are thumbnails and exports, 
thumbnails implementation can change, can be added when we are doing frontend changes.
exports will be required only for csv/excel or pdf exports, can be added when we are supporting exports in platform side.